packages/flutter/lib/src/services/platform_channel.dart

```diff
@@ -11,6 +11,10 @@ import 'binary_messenger.dart';
 import 'message_codec.dart';
 import 'message_codecs.dart';
 
+/// Returns the number of invoke methods waiting for return value.
+int get pendingChannelInvokeMethodCount => _pendingChannelInvokeMethodCount;
```

Comment:
Should this be a static member of MethodChannel instead of being in the global name space?

Reply:
Not familiar with Dart's global variables, but it appears to me that platform_channel may be a more appropriate scope, as this number could possibly count pending invokeMethods in other message channel classes (if later we add more implementations). LMK your thoughts.

Reply:
It's currently documented to only count MethodChannel.invokeMethod, so I think it does belong in there. If in the future we every add new MethodChannels (seems unlikely at this point) we can revisit that.

Reply:
Done. Moved to a static field in MethodChannel.